tests/components/zha/test_gateway.py

```diff
@@ -157,18 +160,24 @@ async def test_gateway_group_methods(hass, device_light_1, device_light_2, coord
 
     # test creating a group with 1 member
     zha_group = await zha_gateway.async_create_zigpy_group(
-        "Test Group", [device_light_1.ieee]
+        "Test Group", [GroupMember(device_light_1.ieee, 1)]
     )
     await hass.async_block_till_done()
 
     assert zha_group is not None
     assert len(zha_group.members) == 1
     for member in zha_group.members:
-        assert member.ieee in [device_light_1.ieee]
+        assert member.device.ieee in [device_light_1.ieee]
 
     # the group entity should not have been cleaned up
     assert entity_id not in hass.states.async_entity_ids(LIGHT_DOMAIN)
 
+    with patch("zigpy.zcl.Cluster.request", side_effect=asyncio.TimeoutError):
+        await zha_group.members[0].async_remove_from_group()
+        assert len(zha_group.members) == 1
+        for member in zha_group.members:
+            assert member.device.ieee in [device_light_1.ieee]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)